### PR TITLE
Extensis | Universal Type Server

### PR DESCRIPTION
**Pull request by djansen1987**
add Extensis | Universal Type Server not Vulnerable after updating to 7.06

Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - Status: please select a value from the status table at the top
  - Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - Please mind the sorting
  - Please put vendor/product name in PR title (instead of "Update README.md")

### CODE REVIEW

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
@djansen1987 It looks like you switched version and status.

**Comment by djansen1987** [comment]
@tschmidtb51  don't know what you mean by that. The brand and product was not yet present on the list

File: software/README.md [hunk 1]
```diff
@@ -767,6 +767,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Exivity | Exivity On-Premise | All version | Not Vuln | | [source](https://docs.exivity.com/getting-started/releases/announcements#announcement-regarding-cve-2021-44228) |
 | Exact | All Products | | Not vuln | | [source NL](https://www.exact.com/nl/nieuws/algemeen-statement-apache-lek), [source EN](https://www.exact.com/news/general-statement-apache-leak) |
 | Evolveum | midPoint || Not vuln | | [source](https://evolveum.com/midpoint-not-vulnerable-to-log4shell/) |
+| Extensis | Universal Type Server | Fix | =>7.0.6 | |[source](https://help.extensis.com/hc/en-us/articles/4412767414299-Universal-Type-Server-7-and-Log4j-vulnerabilities) |
```
Comment (tschmidtb51) [inline software/README.md:770]: The version come first in the table then the status follows:
```suggestion
| Extensis | Universal Type Server | =>7.0.6 | Fix | |[source](https://help.extensis.com/hc/en-us/articles/4412767414299-Universal-Type-Server-7-and-Log4j-vulnerabilities) |
```

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]

File: software/README.md (hunk 1 above)
Comment (tschmidtb51) [reply software/README.md:770]: See also https://github.com/NCSC-NL/log4shell/blob/744f2368c633911d596aee7ba13444a2299aecc0/software/README.md?plain=1#L740

**Comment by djansen1987** [comment]
okay, i understand now. Thanks for the correction!

**Pull request merged**